module-infrastructure/persistence-db/src/main/kotlin/site/yourevents/guest/repository/OwnerNicknameJPARepository.kt

```diff
@@ -0,0 +1,8 @@
+package site.yourevents.guest.repository
+
+import org.springframework.data.jpa.repository.JpaRepository
+import site.yourevents.guest.entity.GuestEntity
+import java.util.UUID
+
+interface OwnerNicknameJPARepository : JpaRepository<GuestEntity, UUID> {
```

Comment:
Guest 엔티티를 담당하는 인터페이스이기에, 다음과 같이 수정해야 할 것 같습니다!
```suggestion
interface GuestJPARepository : JpaRepository<GuestEntity, UUID> {
```

Reply:
수정하였습니다!!